src/main/java/tc/oc/pgm/scoreboard/SidebarMatchModule.java

```diff
@@ -463,6 +463,17 @@ public int compare(Goal a, Goal b) {
           }
         }
       }
+      if (Config.SidebarMessage.enabled()) {
+        if (Config.SidebarMessage.atTop()) {
+          rows.add(0, Config.SidebarMessage.format());
+        } else {
+          if (rows.size() >= MAX_ROWS) {
+            rows.set(MAX_ROWS - 1, Config.SidebarMessage.format());
+          } else {
+            rows.add(Config.SidebarMessage.format());
+          }
+        }
+      }
```

Comment:
I think there should a setting `overwrite-existing` default to `false`. If they scoreboard if full already, we can lose important information about the objectives/match.